exampleCourse/courseInstances/Sp15/assessments/exam02-realTimeGradingDisabled/assessment.html

```diff
@@ -0,0 +1 @@
+For this quiz you can use the <a target="_blank" href="{{ client_files_course }}/formulas.pdf">formula sheet</a>.
```

Comment:
IMO, we could remove this from every assessment except `exam01-bestPractices`, I don't think it serves much value to repeat it so many times.